v0.5/fastn-section/src/parser/section_init.rs

```diff
@@ -7,11 +7,14 @@ pub fn section_init(
     scanner.skip_spaces();
     let name = fastn_section::parser::kinded_name(scanner)?;
     scanner.skip_spaces();
+    let function_marker = scanner.token("()");
```

Comment:
I would first look for token `(`, skip space, and then look for `)`, so even `( )` etc can be parsed.